aws-gitlab-main.yml

```diff
@@ -340,20 +395,31 @@ Resources:
         - !Ref GitLabPrivateSubnetA
         - !Ref GitLabPrivateSubnetB 
 
+  GitLabDBRotationSecrets: 
```

Comment:
Look into Secret for RDS. What you doing now is storing plain secret I think, you can generate RDS secret on deploy and associate with the DB instance blow. 

If not, what you got is fine I think

Reply:
I'm a bit confused. Sounds like GenerateStringKey is generating a secret on deploy with "password" attached at the front of the randomly generated string. I didn't see anything like secrets specific for RDS?

Reply:
https://docs.aws.amazon.com/secretsmanager/latest/userguide/integrating_cloudformation.html
Missing piece:  "Type": "AWS::SecretsManager::SecretTargetAttachment",